src/DataTable/index.jsx

```diff
@@ -1,76 +1,93 @@
-import React from 'react'
-
-import Pagination from './Pagination'
-import Row from './Row'
-import Search from './Search'
+import React, { Fragment } from "react";
+import { getTotalNumberOfPages, search } from "./logic";
+import PaginationBox from "./PaginationBox";
+import Table from "./Table";
+import SearchBox from "./SearchBox";
 
 class DataTable extends React.Component {
   state = {
-    rows: this.props.rows,
-    currentPageNumber: 0,
-    totalNumberOfPages: this.calculateTotalNumberOfPages(this.props.rows)
-  }
-
-  static defaultProps = {
```

Comment:
why do you use static variable, need more explanation. 
default props is not set in this way. 
I haven't use static variables in my React life.